pkg/authz/auth.go

```diff
@@ -53,6 +53,7 @@ type QueryParameterRewriteConfig struct {
 // be used to rewrite a SubjectAccessReview on a given request.
 type HTTPHeaderRewriteConfig struct {
 	Name string `json:"name,omitempty"`
+	Sep  string `json:"separator,omitempty"`
```

Comment:
```suggestion
	Separator  string `json:"separator,omitempty"`
```

NB: I'm not a reviewer here, just adding a suggestion.